webservices/args.py

```diff
@@ -483,10 +483,8 @@ def make_seek_args(field=fields.Int, description=None):
         fields.Str(validate=validate.OneOf(['individual', 'committee'])),
         description='Filters individual or committee contributions based on line number'
     ),
-    'two_year_transaction_period': fields.Int(
+    'two_year_transaction_period': fields.List(fields.Int,
         description=docs.TWO_YEAR_TRANSACTION_PERIOD,
-        required=True,
-        missing=SQL_CONFIG['CYCLE_END_YEAR_ITEMIZED']
```

Comment:
I don't believe this is needed in `config.py` anymore. Can you confirm and remove? 
https://github.com/fecgov/openFEC/blob/develop/webservices/config.py#L27-L29

Reply:
remove the parameter from config.py. cleaned up on sched_a.py and sched_b.py. Thx